### DISCUSSION TITLE
Country List vs Selected (Blocklist/Whitelist) and actions - Question

### DISCUSSION THREAD

**Discussion by TheSimu** [HomeITAdmin/nextcloud_geoblocker#83] [Q&A]
Hi there

I try to use (and make sense of) geoblock and do have the following question:
I chose to mark the country i'm living in and it's surrounding countries as a whitelist resp. as allowed list:
![grafik](https://user-images.githubusercontent.com/86853588/126276796-658409fb-9876-4f53-9f33-226ff5584710.png)
at the bottom of the list, i see the chosen countries (DE, FR, CH, AT)

Action wise: is now allready something active, when this is done? I don't think so... but...
Down below, i see two action-setting like delay login and block login, where the chosen wording is unclear:
When i chose the above countries as allowed-countries (and all other are not allowed):
- will the login be allowed or undelayed for the above mentioned countries (DE. FR, CH, AT) (and all others not?) or not?

Thanks for any enlightening.
BR
TS

**Comment by HomeITAdmin** [comment]
Hello,

The first step is to choose the countries you want to do something for. 
For that you have two different modes: Select the ones you want to do something for (blocklist) or chose all but the ones, that you select in the list (allowlist). So you chose all countries but "DE, AT, CH, FR". 

In the second step you decide what is the reaction when a login attempt from the chosen countries is happening (for the not chosen ones nothing will be done independent of your further selection). 
For these login attempts logging is always active and you can decide, what information is logged (for you it is all). And you can decide if they are delayed and blocked.  (you decided that they are not blocked or delayed).

With the test section at the end of the page you can also test what is happening in the app during login with different IP addresses.

Best

**Reply by TheSimu** [reply]
Thanks a lot for this helpful explanation - i thought "too far" and got it now. Thanks again, BR
